pyproject.toml

```diff
@@ -1,7 +1,44 @@
 [build-system]
-requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
+requires = ["setuptools>=61", "setuptools_scm[toml]>=6.2"]
```

Comment:
Please note that changes in this PR require setuptools>=61. 

[setuptools v61.0.0](https://setuptools.pypa.io/en/latest/history.html#v61-0-0) was released on Mar. 24, 2020. This version starts to [support storing project metadata in `pyproject.toml` following PEP621](https://github.com/pypa/setuptools/issues/1688) but it's still experimental and may change in future versions. I think we should not merge this PR unless the `pyproject.toml` support is stable.

Reply:
Yes, let's wait and see first. I've had to pin to setuptools=58.4.0 on one of my projects 3 weeks ago because one of the dependencies didn't like setuptools v60.9. There might be some other packages pinning to a lower setuptools version too.